### PR TITLE
To Do list project code

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
In this To-Do-List project I implemented the following features for list items:

- Adding a new item.
- Removing a selected item.
- Marking a selected item as complete.
- Removing all items marked as complete at once.
- Reordering a selected item (as drag-and-drop) [as an additional feature].

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+};
+// Add Item in the list when user press enter after input
+const input = document.querySelector('#additem');
+input.addEventListener('keyup', (event) => {
+  if (event.keyCode === 13) { // Returns Enter Key
+    localstore.add(event);
+    event.target.value = '';
+    const oldList = document.querySelectorAll('.todoItem');
+    [...oldList].forEach((e) => e.remove());
+    renderUI();
```
Comment (nedith) [inline src/index.js:53]: You are repeating this code three times in this file. I suggest you create a function and call it when required

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    });
+  });
+  // Trash task from list
+  const trashes = document.querySelectorAll('.fa-trash');
+  [...trashes].forEach((trash) => {
+    trash.addEventListener('click', (event) => {
+      localstore.remove(event);
+      const oldList = document.querySelectorAll('.todoItem');
+      [...oldList].forEach((e) => e.remove());
+      renderUI();
```
Comment (nedith) [inline src/index.js:41]: You are repeating this code two other times in this file and therefore, violating the **DRY** principle. Kindly check out this article [DRY, KISS and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) if you are unsure how to proceed.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }
+});
+// Remove Completed Tasks
+const btn = document.querySelector('.btn');
+btn.addEventListener('click', (e) => {
+  e.preventDefault();
+  localstore.clearCompleted();
+  const oldList = document.querySelectorAll('.todoItem');
+  [...oldList].forEach((e) => e.remove());
+  renderUI();
```
Comment (nedith) [inline src/index.js:63]: Kindly create a function as suggested earlier to avoid this repetition as the _**DRY**_ principles suggest

**Review by nedith — CHANGES_REQUESTED** [review]
Hi, @saqibjavaiddev :hand: 
**Good job on implementing the full functionality of the project** :+1: 
However, there is a minor issue to fix in order to make your code cleaner.
<div id="header" align="center">
 
</div>

### Highlights

- Following best JS coding practices
- No Linter Errors
-  Using ES6 functionality
### Changes Required :recycle: 
_Please check the comments under the review for more details_

**Comment by saqibjavaiddev** [comment]
Hello, @nedith Thanks for the review. I am working on it and will fix that soon

**Pull request merged**